### DISCUSSION TITLE
[Question] Difference between "Extra variables" and "Environment variables" ?

### DISCUSSION THREAD

**Discussion by Laurent-1971** [semaphoreui/semaphore#1476] [Q&A] [answered]
Hello.
When creating an environment, I don't undestand the difference between "Extra variables" and "Environment variables". 
Doc says nothing about it.

**Comment by ansibleguy** [comment, marked as answer]
Greetings!

As I understand it:
* Environment variables => are set as actual [environment variables](https://en.wikipedia.org/wiki/Environment_variable)

  You can p.e. use them to [override Ansible-Config](https://docs.ansible.com/ansible/latest/reference_appendices/config.html)

* Extra variables => are passed to Ansible using the [--extra-vars flag](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#defining-variables-at-runtime)
